### DISCUSSION TITLE
Filter and brush time doesn't appear

### DISCUSSION THREAD

**Discussion by brunoper91** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#424] [Q&A] [answered]
on my card the time from Filters and brushes doesn't appear

![filtros](https://user-images.githubusercontent.com/58269980/176169377-c011b3b2-c4d8-44fd-87a0-46b4b19637f5.jpg)

some help ?

**Comment by PiotrMachowski** [comment]
Some config/details? 😉

**Comment by brunoper91** [comment, marked as answer]
i did a integration with miio, after that I used the card

as you can see, everything works, only the hours of the filters and brushes don't appear, but with the costum:card doesn't appear either...
![filtros2](https://user-images.githubusercontent.com/58269980/176210544-511bccbc-f2de-4966-96c1-d575b3b3d367.jpg)

**Reply by PiotrMachowski** [reply]
Change vacuum_platform to `default`

**Reply by brunoper91** [reply]
it worked thx :D
